### DISCUSSION TITLE
无法安装“地震预警”插件

### DISCUSSION THREAD

**Discussion by AWDNDWindowsXP** [ClassIsland/ClassIsland#493] [Q&A] [answered]
问题

我发现安装该插件后，Classisland提示“重启以应用更改”，重启后依然提示，且无法删除该插件。
临时解决方案

打开ClassIsland\Cache\PluginPackages目录，删掉denglihong2007.EarthquakeWarning.cipx
重启Classisland

**Comment by jiangyin14** [comment]
该插件可能未完成开发。
如果希望获得社区支持，至少需要把插件的文件提供一下。

且本问题不应在ClassIsland仓库进行反馈，建议前往对应插件的仓库或联系开发负责人。

**Reply by Griseo-bh3rd** [reply]
已经上架了，我这里也没法安装

**Reply by AWDNDWindowsXP** [reply]
> 该插件可能未完成开发。 如果希望获得社区支持，至少需要把插件的文件提供一下。
> 
> 且本问题不应在ClassIsland仓库进行反馈，建议前往对应插件的仓库或联系开发负责人。

[下载链接](https://github.com/denglihong2007/EarthquakeWarningForClassIsLand/releases)

**Comment by HelloWRC** [comment, marked as answer]
可能是这个插件的打包格式有问题，manifest.yml等内容应该直接放置在根目录，而这个插件的作者把插件内容包含到了一个子文件夹下（如图）。请联系插件作者以解决这个问题。

![图片](https://github.com/user-attachments/assets/dd9eb054-af73-441a-844b-c7b71130eec9)
